src/update/lib/routers/report.js

```diff
@@ -14,9 +14,17 @@ const {
 const reportsDir = path.resolve(__dirname, '..', '..', 'data', 'reports');
 
 /**
- * get the most recent report in JSON format
+ * Route that give the list of reports or the content of most recent report in JSON format.
  *
- * @return report
+ * @param {Object} req - HTTP request.
+ * @param {Object} res - HTTP response.
+ *
+ * @routeQuery {String} latest - indicate if it gave latest report or not.
+ *
+ * @routeResponse {Array<String>|<Object>} List of filename of report
```

Comment:
Not the right way to define an union :
```suggestion
 * @routeResponse {Array<string|Object>} List of filename of report
```
or
```suggestion
 * @routeResponse {Array<string>|Array<Object>} List of filename of report
```